package.json

```diff
@@ -1,46 +1,39 @@
 {
-  "name": "bucklescript-tea",
-  "version": "0.15.0",
-  "description": "TEA for Bucklescript",
-  "main": "index.js",
-  "scripts": {
-    "clean": "bsb clean",
-    "clean:all": "bsb -clean-world",
-    "build": "bsb",
-    "build:all": "bsb -make-world",
-    "watch": "bsb -w",
-    "build:test:client": "browserify --standalone=Test_client --outfile=lib/js/test-ocaml/app_test_client.js lib/js/test-ocaml/test_client.js",
-    "watch:test:client": "watchify --standalone=Test_client --outfile=lib/js/test-ocaml/app_test_client.js lib/js/test-ocaml/test_client.js",
-    "watch:test": "run-p watch watch:test:*",
-    "prebuild:test": "run-s build",
-    "build:test": "run-p build:test:*",
-    "pretest": "run-s build:test",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "dual-syntax": "bin/dual-syntax"
-  },
+  "name": "rescript-tea",
+  "version": "0.6.1",
```

Comment:
I'm not sure why this file changes, it should be pretty much the same as it was before (with some adaptations for rescript)